### PR TITLE
New Profile - Diamond Nodes

### PR DESCRIPTION
**Pull request by DiamondNodes**
New Validator Profile for active set validator "Diamond Nodes"

Any changes required please let us know.

### CODE REVIEW

**Comment by terran6** [comment]
@DiamondNodes Hello!

Is Stake n Squeeze your original profile?

Just ensuring that you have the authorization to remove their information.

Thanks.

**Comment by DiamondNodes** [comment]
Correct please delete.

File: terravaloper1fvj38kgxhf3hqa8yxrartuguqjd4er5jfmmf7y/profile.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "contact": {
+    "website": "https://diamondnodes.io",
+    "email": "hello@diamondnodes.io",
+    "twitter": "@_DiamondNodes",
+    "discord": "Diamond Nodes#3413",
+    "github": "DiamondNodes" ,
```
Comment (terran6) [inline terravaloper1fvj38kgxhf3hqa8yxrartuguqjd4er5jfmmf7y/profile.json:7]: @DiamondNodes Please remove the trailing comma after your Github username and then we can merge.  Thanks!

**Review by terran6 — CHANGES_REQUESTED** [review]

File: terravaloper1fvj38kgxhf3hqa8yxrartuguqjd4er5jfmmf7y/profile.json (hunk 1 above)
Comment (DiamondNodes) [reply terravaloper1fvj38kgxhf3hqa8yxrartuguqjd4er5jfmmf7y/profile.json]: Sorry about that, done!

**Review by terran6 — APPROVED** [review]

**Pull request merged**